src/main/java/seedu/address/model/person/Gender.java

```diff
@@ -0,0 +1,57 @@
+package seedu.address.model.person;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a Person's Gender in the address book.
+ * Guarantees: immutable; is valid as declared in {@link #isValidGender(String)}
+ */
+public class Gender {
+
+    public static final String MESSAGE_CONSTRAINTS = "Gender can take any values, and it should not be blank";
+
+    /*
+     * The first character of the address must not be a whitespace,
+     * otherwise " " (a blank string) becomes a valid input.
+     */
+    public static final String VALIDATION_REGEX = "[^\\s].*";
```

Comment:
Should we limit Gender to take only some values e.g. F and M?

Reply:
At least for the first version I think we should not make it too restrictive. I will restrict it in future versions